scripts/dss-ops.py

```diff
@@ -9,9 +9,11 @@
 sys.path.insert(0, pkg_root)  # noqa
 
 import dss
+import dss.operations.storage
 from dss.operations import dispatch
 
 dss.Config.set_config(dss.BucketConfig.NORMAL)
 
 if __name__ == "__main__":
+    print(dispatch.job_id)
```

Comment:
Should this have more context?  Consider: `print(f'Dispatch Job ID: {dispatch.job_id}')`.

Reply:
fixed